packages/autocomplete/src/index.ts

```diff
@@ -78,7 +80,7 @@ const GROUP = '$group';
  *
  * @returns {array} Matching Completions, Current User Input.
  */
-async function completer(
+export async function completer(
```

Comment:
I removed the default export, so you now either use `completer` (the old one) or `initNewAutocompleter` (the new one).

I know having "new" in there and having them asymmetrical like that is a bit ugly. Open to suggestions. I can go through these things and rename everything to old and new or we can live with the fact that this is temporary and clean it up when we remove the old one? Probably various other options.

Reply:
This is fine, removing default exports is a good idea anyway